src/main/java/ch/uzh/ifi/seal/soprafs20/controller/UserController.java

```diff
@@ -53,4 +53,52 @@ public UserGetDTO createUser(@RequestBody UserPostDTO userPostDTO) {
         // convert internal representation of user back to API
         return DTOMapper.INSTANCE.convertEntityToUserGetDTO(createdUser);
     }
+
+    @GetMapping("/user/{userId}")
+    @ResponseStatus(HttpStatus.OK)
+    @ResponseBody
+    public UserGetDTO getUserByUserName(@RequestHeader("X-Auth-Token") String token, @PathVariable Long userId){
+        UserGetDTO userGetDTO = new UserGetDTO();
+        return  userGetDTO;
+
+    }
+
+    @GetMapping("/user/login")
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    @ResponseBody
+    public UserGetDTO login(@RequestBody UserPostDTO userPostDTO){
```

Comment:
Can you implement a UserAuthDTO that is returned here, containing just the Token?

Reply:
Done.